tuning/automation/rocblas-parser.py

```diff
@@ -0,0 +1,367 @@
+                gemmconfig +=f" call_count: {size[8]}  {duplicate}\n"
+            else:
+                gemmconfig +=f"  {duplicate}\n"
+
+        f.write(gemmconfig)
+        previous_size = size[:-2]
+
+    f.write(f"\n -- total unique is: {count_unique}")
+
+def create_rocBLAS_bench(benchFile, matrices,verify,initialization):
+
+    bench = []
+
+    for gemmtype in matrices:
+        print(gemmtype)
+        # get GEMM function type: a/b/c/d/compute_type
+        problemDict = {}
+        if ("HHS" in gemmtype):
+            problemDict["rocblas_function"] = "rocblas_gemm_strided_batched_ex" if "SB" in gemmtype else  "rocblas_gemm_ex"
+            problemDict["a_type"] = "f16_r"
+            problemDict["b_type"] = "f16_r"
+            problemDict["c_type"] = "f16_r"
+            problemDict["d_type"] = "f16_r"
+            problemDict["compute_type"] = "f32_r"
+    
+        elif ("HSS" in gemmtype):
+            problemDict["rocblas_function"] = "rocblas_gemm_strided_batched_ex" if "SB" in gemmtype else  "rocblas_gemm_ex"
+            problemDict["a_type"] = "f16_r"
+            problemDict["b_type"] = "f16_r"
+            problemDict["c_type"] = "f32_r"
+            problemDict["d_type"] = "f32_r"
+            problemDict["compute_type"] = "f32_r"
+
+        elif ("BBS" in gemmtype):
+            problemDict["rocblas_function"] = "rocblas_gemm_strided_batched_ex" if "SB" in gemmtype else  "rocblas_gemm_ex"
+            problemDict["a_type"] = "bf16_r"
+            problemDict["b_type"] = "bf16_r"
+            problemDict["c_type"] = "bf16_r"
+            problemDict["d_type"] = "bf16_r"
+            problemDict["compute_type"] = "f32_r"
+    
+        elif ("BSS" in gemmtype):
+            problemDict["rocblas_function"] = "rocblas_gemm_strided_batched_ex" if "SB" in gemmtype else  "rocblas_gemm_ex"
+            problemDict["a_type"] = "bf16_r"
+            problemDict["b_type"] = "bf16_r"
+            problemDict["c_type"] = "f32_r"
+            problemDict["d_type"] = "f32_r"
+            problemDict["compute_type"] = "f32_r"
+
+        elif ("SGEMM" in gemmtype):
+            problemDict["rocblas_function"] ="rocblas_sgemm_strided_batched" if "SB" in gemmtype else "rocblas_sgemm"
+
+        elif ("DGEMM" in gemmtype):
+            problemDict["rocblas_function"] ="rocblas_dgemm_strided_batched" if "SB" in gemmtype else "rocblas_dgemm"
+
+        # get GEMM function and matrix orientation: transA/B, a/b/c/d/compute_type
+        if ("NN" in gemmtype):
+            problemDict["transA"] = "N"
+            problemDict["transB"] = "N"
+        elif ("NT" in gemmtype):
+            problemDict["transA"] = "N"
+            problemDict["transB"] = "T"
+        elif ("TN" in gemmtype):
+            problemDict["transA"] = "T"
+            problemDict["transB"] = "N"
+        elif ("TT" in gemmtype):
+            problemDict["transA"] = "T"
+            problemDict["transB"] = "T"
+
+        if "SB" in gemmtype:

```

Comment:
Can we make default iteration smaller?
Plus, please make number of iteration for DB same as SB.

Reply:
I thought SB was SGEMM.
Looks like SB is stridedBatched....
Anyway, I would like to make iteration number smaller by default.